src/renderer/hooks/useBackendEventSource.ts

```diff
@@ -28,6 +28,7 @@ export interface BackendEventMap {
         nodeId: string;
         executionTime?: number | null;
         data?: OutputData | null;
+        progressPercent?: number;
```

Comment:
There's the problem. Wrong type. Python's None gets jsonified to null.
```suggestion
        progressPercent?: number | null;
```